pkg/network/iperf/iperf.go

```diff
@@ -0,0 +1,71 @@
+package iperf
+
+import (
+	"os/exec"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/zos/pkg/zinit"
+)
+
+const (
+	zinitService = "iperf"
+)
+
+// IPerfServer represent an iperf server
+type IPerfServer struct {
```

Comment:
I will replace all of this with one function `Ensure` that checks if the zinit unit file for iperf exists, and if not add one and call monitor and that's it. Other functions are not needed (and can be then added later if needed, but don't see why would thay) 
then a caller can directly call `iperf.Ensure` where iperf is the package name (Ensure is a function not an instance method) no need to have an empty struct